README.md

```diff
@@ -58,6 +65,21 @@ vars:
 ## (Optional) Step 5: Additional configurations
 <details><summary>Expand to view configurations</summary>
 
+### Passing Through Additional Fields
+This package includes all source columns defined in the macros folder. You can add more columns using our pass-through column variables. These variables allow for the pass-through fields to be aliased (`alias`) and casted (`transform_sql`) if desired, but not required. Datatype casting is configured via a sql snippet within the `transform_sql` key. You may add the desired sql while omitting the `as field_name` at the end and your custom pass-though fields will be casted accordingly. Use the below format for declaring the respective pass-through variables:
+
+```yml
+vars:
+    recurly_account_pass_through_columns: 
+      - name: "new_custom_field"
+        alias: "custom_field"
+        transform_sql: "cast(custom_field as string)"
+      - name: "another_one"
+    recurly_subscription_pass_through_columns:
+      - name: "this_field"
+        alias: "cool_field_name"
+```
+
```

Comment:
Since you added the pass through variables, we should document them as well.

Reply:
Makes sense!